Prog/Fix NL000093 and NL000319.sas

```diff
@@ -0,0 +1,424 @@
+                   CLUSTER_ $
+                   POLDIST $
+                   ROC $
+                   CENSUS_TRACT
+                   VOTE_PRCNCT $
+                   SMD $
+                   ZIPCODE
+                   NATIONALGRID $
+                   ROADWAYSEGID
+                   FOCUS_IMPROVEMENT_AREA $
+                   HAS_ALIAS $
+                   HAS_CONDO_UNIT $
+                   HAS_RES_UNIT $
+                   HAS_SSL $
+                   LATITUDE
+                   LONGITUDE
+                   STREETVIEWURL $
+                   RES_TYPE $
+                   WARD_2002 $
+                   WARD_2012 $
+                   ANC_2002 $
+                   ANC_2012 $
+                   SMD_2002 $
+                   SMD_2012 $
+                   MARID
+                   IMAGEURL $
+                   IMAGEDIR
+                   IMAGENAME $
+                   CONFIDENCELEVEL $
+       ;
+       if _ERROR_ then call symputx('_EFIERR_',1);  /* set ERROR detection macro variable */
+       run;
+
+filename fimport clear;
+
+  
+proc sort data=Mar_address;
+  by address_id;
+
+**Merge project info and address info for new projects**;
+
+data bldg_mar_address;
+
+  set Mar_address (rename = (address_id = bldg_address_id fulladdress=bldg_addre Streetviewurl=Bldg_streetview_url));
+  
+  format _all_ ;
+  informat _all_ ;
+  
+  
+  ** Standard geos **;
+  
+  length Ward2012 $ 1;
+  
+  Ward2012 = substr( Ward_2012, 6, 1 );
+  
+  format Ward2012 $ward12a.;
+  
+  length Anc2012 $ 2;
+  
+  Anc2012 = substr( Anc_2012, 5, 2 );
+  
+  format Anc2012 $anc12a.;
+  
+  length Psa2012 $ 3;
+  
+  Psa2012 = substr( Psa, 21, 3 ); 
+  
+  format Psa2012 $psa12a.;
+  
+  length Geo2010 $ 11;
+  
+  if Census_tract ~= . then Geo2010 = "11001" || put( Census_tract, z6. );
+  
+  format Geo2010 $geo10a.;
+  
+  length Cluster_tr2000 $ 2 Cluster_tr2000_name $ 80;
+  
+  if Cluster_ ~= "" then Cluster_tr2000 = put( 1 * substr( Cluster_, 9, 2 ), z2. );
+  
+  format Cluster_tr2000 $clus00a.;
+  
+  length Zip $ 5;
+  
+  Zip = put( ZIPCODE, z5.0 );
+  
+  format Zip $zipa.;
+
+  ** Cluster names **;
+  
+  Cluster_tr2000_name = put( Cluster_tr2000, $clus00b. );
+  
+  ** Image url **;
+  
+  if imagename ~= "" and imagename ~=: "No_Image_Available" then 
+    bldg_image_url = trim( imageurl ) || "/" || trim( left( imagedir ) ) || "/" || imagename;
+  
+run;
+
+data Building_Geocode;
+set prescat.Building_Geocode;
+run;
+
+data Building_Geocode;
+modify Building_Geocode bldg_mar_address;
+by nlihc_ID;
+run;
+
+
+/*proc sort data=building_geocode;
+ by nlihc_id Bldg_addre;
+ run;*/
+
+title2 '********************************************************************************************';
+title3 '** 3/ Check for changes in building_geocode file unrelated to fixes for NL000093 and NL000319';
+
+proc compare base=prescat.building_geocode compare=building_geocode listall maxprint=(40,32000);
+ id nlihc_id proj_name;
```

Comment:
ID statement needs to include Bldg_addre. Otherwise, observations are not identified uniquely.

> WARNING: The data set PRESCAT.BUILDING_GEOCODE contains a duplicate observation at observation number 5.
> NOTE: At observation 5 the current and previous ID values are:
>       NLIHC_ID=NL001008 Proj_Name=TCB Fairlawn Marshall.